src/btree/bt_delete.c

```diff
@@ -399,11 +403,12 @@ __wt_delete_page_instantiate(WT_SESSION_IMPL *session, WT_REF *ref)
 
                     if (page_del != NULL)
                         page_del->update_list[count++] = upd;
+                    total_size += size;
```

Comment:
why the total_size is incremented here. line 396 represents the memory allocation at 390. I am not able to find the corresponding memory allocation for this increment.

Reply:
There is an another memory allocation that happens at line 400.